src/stories/Components.Badge.stories.tsx

```diff
@@ -0,0 +1,49 @@
+// Button.stories.ts|tsx
```

Comment:
```suggestion
// Badge.stories.ts|tsx
```

Reply:
Yes you are right, i'll fix this in the next commit